### ISSUE TITLE
Epic Games Store instantly crashes, am I missing something?

### ISSUE THREAD

**Issue by Nero-deb** [bottlesdevs/programs#52]
Epic Games Store instantly crashes.

I have freshly installed flatpak Bottles with a Gaming bottle.

Do I need to have anything specific installed for Bottles to run properly? I cannot find any information. I installed Wine from Gnome Software, but I've no idea whether I have to do anything else.

I'm on Zorin OS.

I couldn't find anything anywhere about what exactly Bottles needs to run correctly. This is my first experience with Bottles. :(

**Comment by Nero-deb** [comment]
> Can you attach the output? Start it from terminal to pick it up.

I sadly have no idea how to run Bottles from the terminal. I assume it should go like `flatpak run [x]` but bottles doesn't work and it says the name must contain at least 2 periods, however, I cannot find what it ought to be.

**Comment by mirkobrombin** [comment]
`flatpak run com.usebottles.bottles`

**Comment by Nero-deb** [comment]
> `flatpak run com.usebottles.bottles`

`$ flatpak run com.usebottles.bottles
Gtk-Message: 13:03:29.132: Failed to load module "canberra-gtk-module"
(13:03:42) INFO Performing Bottles checks... 
(13:03:42) INFO Dxvks found:
	 - dxvk-1.10.1
 
(13:03:42) INFO Vkd3ds found:
	 - vkd3d-proton-2.6
 
(13:03:42) INFO Nvapis found:
	 - dxvk-nvapi-v0.5.3
 
(13:03:42) INFO Latencyflexs found:
	 - latencyflex-v0.1.0
 
(13:03:43) INFO Runners found:
	 - sys-wine-7.0
	 - caffe-7.5
 
(13:03:44) INFO Bottles found:
	 - GAMER
 
(13:03:45) INFO Bottles Started! 
`

Is this what you meant?

**Comment by mirkobrombin** [comment]
Yes but maybe launch the game too :D

**Comment by Nero-deb** [comment]
> Yes but maybe launch the game too :D

Sorry!

This is when I try to launch Epic Games Store:

> ~$ flatpak run com.usebottles.bottles
> Gtk-Message: 13:11:13.230: Failed to load module "canberra-gtk-module"
> (13:11:14) INFO Performing Bottles checks... 
> (13:11:14) INFO Dxvks found:
> 	 - dxvk-1.10.1
>  
> (13:11:14) INFO Vkd3ds found:
> 	 - vkd3d-proton-2.6
>  
> (13:11:14) INFO Nvapis found:
> 	 - dxvk-nvapi-v0.5.3
>  
> (13:11:14) INFO Latencyflexs found:
> 	 - latencyflex-v0.1.0
>  
> (13:11:14) INFO Runners found:
> 	 - sys-wine-7.0
> 	 - caffe-7.5
>  
> (13:11:14) INFO Bottles found:
> 	 - GAMER
>  
> (13:11:15) INFO Bottles Started! 
> (13:11:17) INFO Setting Key: [discrete_gpu] to [True] for bottle: [GAMER]… 
> (13:11:17) INFO Setting Key: [pulseaudio_latency] to [True] for bottle: [GAMER]… 
> (13:11:17) WARNING Cannot find states.yml file for bottle: [GAMER] 
> (13:11:17) INFO Using WINE debug tool -- get_processes 
> (13:11:17) INFO Using WINE debug tool -- get_processes 
> 
> (bottles:2): Handy-CRITICAL **: 13:11:17.190: hdy_stackable_box_set_visible_child_name: assertion 'contains_child' failed
> esync: up and running.
> 0034:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -r" (2).
> (13:11:18) INFO Using WINE debug tool -- get_processes 
> (13:11:18) INFO Using WINE debug tool -- get_processes 
> (13:11:19) INFO Using WINE debug tool -- get_processes

**Comment by mirkobrombin** [comment]
I don't see the launch

**Comment by mirkobrombin** [comment]
This is what happen when you start a program:
```
(14:15:45) INFO Launching an executable… 
(14:15:45) WARNING Windows path detected. Avoiding validation. 
(14:15:45) INFO Using Bottles runtime 

```

**Comment by Nero-deb** [comment]
If I launch again after the start button turns to stop and then back to start again, I get the popup about Epic detecting it crashed previously and then the terminal says the following:

> (13:14:09) INFO Launching an executable… 
> (13:14:09) WARNING Windows path detected. Avoiding validation. 
> esync: up and running.
> 002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -r" (2).
> 0100:err:winediag:gnutls_process_attach Compiled without DH support.
> 0100:err:wldap32:DllMain No libldap support, expect problems
> 017c:err:winediag:gnutls_process_attach Compiled without DH support.
> 017c:err:wldap32:DllMain No libldap support, expect problems
> (13:15:15) INFO Launching an executable… 
> (13:15:15) WARNING Windows path detected. Avoiding validation. 
> esync: up and running.
> 002c:err:wineboot:process_run_key Error running cmd L"C:\\windows\\system32\\winemenubuilder.exe -r" (2).
> 0100:err:winediag:gnutls_process_attach Compiled without DH support.
> 0100:err:wldap32:DllMain No libldap support, expect problems
> 017c:err:winediag:gnutls_process_attach Compiled without DH support.
> 017c:err:wldap32:DllMain No libldap support, expect problems

I don't get INFO Launching on first launch. Only on consecutive after it fails to launch on the first try and then launches the detected failing to launch.

**Comment by mirkobrombin** [comment]
Switch to the runner Caffe 7.2

**Comment by Nero-deb** [comment]
I tried installing the dependencies from: https://www.epicgames.com/help/en-US/epic-games-store-c73/error-codes-c100/how-do-i-fix-the-epic-games-launcher-when-i-get-the-error-the-necessary-prerequisites-have-failed-to-install-a3511

I tried uninstalling and installing Wine in case there were any issues with it.

I tried enabling Bottles runtime, but it didn't help.

I tried enabling VKD3D but it also didn't help.

It still doesn't work. I have no idea why. I really wanted to try Bottles where Steam's Proton doesn't reach, alas, my brain is too small for this.
